docs/sources/administration/correlations/create-a-new-correlation/index.md

```diff
@@ -13,6 +13,25 @@ weight: 40
 
 Make sure you have permission to add new correlations. Only users with write permissions to data sources can define new correlations.
 
+## Create a correlation in Explore's Correlations Editor
+
+1. Go to Explore page.
+1. Select a data source that you would like to be [the source data source]({{< relref "/docs/grafana/latest/administration/correlations/correlation-configuration#source-data-source-and-result-field" >}}) for a new correlation.
+1. Run a query producing data in [a supported visualization]({{< relref "/docs/grafana/latest/administration/correlations/#correlations" >}}).
+1. Click "+ Add" button in the top toolbar and select "Add correlation" (you can also select "Correlations Editor" from the [Command Palette]({{< relref "/docs/grafana/latest/search/#command-palette" >}})).
+1. Explore is now in Correlations Editor mode indicated by a blue border and top bar. You can exit Correlations Editor using the Exit button in the top bar.
+1. The visualization is enriched with links allowing to create new correlations. Links are attached to the data that you can use to build a new query:
+   - Logs: links are displayed next to field values inside log details for each log row
+   - Table: every table cell is a link
+1. Click on a link to begin adding new correlation. Links are associated with a field that will be used as [a results field of a correlation]({{< relref "/docs/grafana/latest/administration/correlations/correlation-configuration" >}}).
```

Comment:
The idea of the results field is a little obscure. I would leave the language you have, but maybe add "This means the correlation link will be created on or by the link you select once the correlation is saved."

Reply:
Added in 874abff3d6ff79e371ed3c1ef082d4f22e7b3531